### ISSUE TITLE
Different test results when logging training accuracy

### ISSUE THREAD

**Issue by blazejdolicki** [facebookresearch/LASER#141]
I really liked the paper and I'm glad the code is published, but I have one minor problem.
When evaluating the model on a text classification task, I wanted to print not only validation accuracy but also training acc.  To do that I added a line "corr_train, nbex_train = net.TestCorpus(train_loader, 'Train')" before line 247 in sent_classif.py. Surprisingly enough, after adding that line the results (on the test set) has changed. I am new to Pytorch, so I'm probably overlooking something, but I can't understand why adding this line has such impact. Would be great to hear if you have any idea @hoschwenk
